config/config.go

```diff
@@ -50,19 +50,22 @@ func (conf *BurrowConfig) Kernel(ctx context.Context, restoreDump string) (*core
 	if conf.ValidatorAddress == nil {
 		return nil, fmt.Errorf("no validator address provided, cannot make Kernel")
 	}
+
 	logger, err := lifecycle.NewLoggerFromLoggingConfig(conf.Logging)
```

Comment:
I think we should probably drop this from being a method of `BurrowConfig` and move it to core in a separate file adjacent to kernel.go as

`ConfigureKernel(conf *BurrowConfig)`